src/sentry/web/frontend/help_platform_base.py

```diff
@@ -15,8 +15,11 @@ def get_project_list(self, user):
             Q(organization__member_set__has_global_access=True, organization__member_set__user=user)
             | Q(team__organizationmember__user=user)
         ).exclude(
-            team__organizationmemberteam__is_active=False,
-        ).select_related('team', 'organization').order_by('organization', 'team').distinct())
+            team__id__in=OrganizationMemberTeam.objects.filter(
```

Comment:
I need to confirm but I **think** this might error if the resulting Team query has no results